AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Models/ReleaseNotesBuilder.swift

```diff
@@ -41,9 +42,7 @@ struct ReleaseNotesBuilder {
     }
 
     func buildServiceChangeSection() throws -> [String] {
-        let features = try featuresReader.getFeaturesFromFile()
-        let mapping = try featuresReader.getFeaturesIDToServiceNameDictFromFile()
-        return buildServiceFeatureSection(features, mapping) + buildServiceDocSection(features, mapping)
+        return buildServiceFeatureSection(features, featuresIDToServiceName) + buildServiceDocSection(features, featuresIDToServiceName)
```

Comment:
Here, the injected features & mapping are used directly to generate the release notes content.